package.json

```diff
@@ -71,13 +71,12 @@
     "@typescript-eslint/eslint-plugin": ">=4.0.1"
   },
   "devDependencies": {
-    "@commitlint/cli": "11.0.0",
-    "@commitlint/travis-cli": "11.0.0",
```

Comment:
Don't remove this package, because it's still necessary. It's removal should be part of #530.

Reply:
Sure, done! :smile: